client/pkg/omnictl/download.go

```diff
@@ -58,6 +63,9 @@ func init() {
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.labels, "initial-labels", nil, "Bake initial labels into the generated installation media")
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.extraKernelArgs, "extra-kernel-args", nil, "Add extra kernel args to the generated installation media")
 	downloadCmd.Flags().StringArrayVar(&downloadCmdFlags.extensions, "extensions", nil, "Generate installation media with extensions pre-installed")
+	downloadCmd.Flags().StringVar(&downloadCmdFlags.siderolinkGRPCTunnelMode, "siderolink-grpc-tunnel-mode",
```

Comment:
Maybe just add a bool flag?
I understand the idea behind AUTO mode, but maybe we can make it work like that:
- no flag set: auto mode.
- flag set to false: override mode to false.
- flag set to true: override mode to true.

Something like pointer to bool could work here, but I'm not sure if flags lib supports that.

Reply:
Not having to remember constants for different modes is better I think.

Reply:
did the change to work exactly that way